db/schema.rb

```diff
@@ -23,6 +23,35 @@
 
   add_index "identities", ["user_id"], :name => "index_identities_on_user_id"
 
+  create_table "items", :force => true do |t|
+    t.integer  "user_id"
+    t.integer  "meal_id"
+    t.string   "name"
+    t.float    "price"
+    t.string   "picture_url"
```

Comment:
`picture_url` is still in here, so you probably didn't reload the schema after you edited your migrations.